components/datetime/src/format/datetime.rs

```diff
@@ -161,9 +161,12 @@ where
             input!(related_iso = year.related_iso());
 
             // Always in latin digits according to spec
-            FixedDecimal::from(related_iso)
-                .padded_start(l.to_len() as i16)
-                .write_to(w)?;
+            {
+                let mut num = SignedFixedDecimal::from(related_iso);
+                num.pad_start(l.to_len() as i16);
+                num
+            }
```

Comment:
Issue: Why did you change `padded_start` to `pad_start`?

However, you'll need the `{ ... }` block anyway if SignedFixedDecimal doesn't duplicate the functions of the inner type. Maybe we can just duplicate the `-ed` functions.

We discussed the `-ed` functions in June: https://github.com/unicode-org/icu4x/issues/2902#issuecomment-2153121175